### ISSUE TITLE
PowerShell New-Object Exception

### ISSUE THREAD

**Issue by lehitoskin** [FLIF-hub/FLIF#403]
Running the new `dl_make_vs.ps1` comes up with this error when downloading zlib:
```
New-Object : Exception calling ".ctor" with "3" argument(s): "End of Central Directory record could not be found."
At
C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules\Microsoft.PowerShell.Archive\Microsoft.PowerShell.Archive.psm1:933
char:23
+ ... ipArchive = New-Object -TypeName System.IO.Compression.ZipArchive -Ar ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [New-Object], MethodInvocationException
    + FullyQualifiedErrorId : ConstructorInvokedThrowException,Microsoft.PowerShell.Commands.NewObjectCommand
```

If I run `dl_make_vs.bat`, it will download and extract zlib correctly.

**Comment by lehitoskin** [comment]
Seems to have been resolved in 862e5c5ae3f596197365d1510f6c0eb251522ef0

**Comment by lehitoskin** [comment]
Oh, wait, I lied. I forgot to clean the directory first and jumped the gun because it compiled.

To reiterate: on a clean directory, the PowerShell script fails on downloading and extracting zlib.

**Comment by SaschaNaz** [comment]
That's very strange, it says that [your zip file is corrupted](http://stackoverflow.com/questions/20960403/c-sharp-end-of-central-directory-record-could-not-be-found).

**Comment by SaschaNaz** [comment]
Quick fix is done here: #405
